style/style.css

```diff
@@ -0,0 +1,239 @@
+/*@media only screen and (max-width: 600px) {
+  body {
+    background-color: #F9F9F9;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-size: 11px;
+  }
+}*/
```

Comment:
- [ ] Although I commend your efforts to use media queries, please remove commented code. It is bad practice to leave commented code in your project. If you need to see your past code, you can check your commit history. 
Screenshot from pathwright 👇 
![Screen Shot 2020-07-14 at 1 09 34 AM](https://user-images.githubusercontent.com/48804258/87362352-af849e80-c56e-11ea-8d96-20ee51b0e53b.png)